src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/TaskManagementService.java

```diff
@@ -124,6 +123,7 @@ public Task getTask(String taskId, AccessControlResponse accessControlResponse)
             TaskResource taskResource = roleAssignmentVerification(taskId, accessControlResponse, permissionsRequired);
             return cftTaskMapper.mapToTask(taskResource);
         } else {
+            Map<String, CamundaVariable> variables = camundaService.getTaskVariables(taskId);
```

Comment:
is there anything defensive that needs doing here? Perhaps a test to prove the behaviour is now as desired, and possibly a line of logging

Reply:
tests were already there to cover this, changing scope of camunda varaibles to fix the bug